packages/client/hmi-client/src/components/workflow/tera-simulation-workflow.vue

```diff
@@ -344,16 +344,16 @@ const testOperation: Operation = {
 const models = computed<Model[]>(() => props.project.assets?.models ?? []);
 const datasets = computed<Dataset[]>(() => props.project.assets?.datasets ?? []);
 
-async function selectModel(node: WorkflowNode, data: { id: string }) {
-	droppedAssetId.value = null;
-	node.state.modelId = data.id;
-
+const refreshModelNode = async (node: WorkflowNode) => {
 	// FIXME: Need additional design to work out exactly what to show. June 2023
-	const configurationList = await getModelConfigurations(data.id);
-	node.outputs = [];
+	const configurationList = await getModelConfigurations(node.state.modelId);
 	configurationList.forEach((configuration) => {
 		// Only add new configurations
-		if (node.outputs.find((d) => isEqual(d.value, [configuration.id]))) return;
+		const existingConfig = node.outputs.find((d) => isEqual(d.value, [configuration.id]));
```

Comment:
What does `d` represent, can you set a variable name that is more explicit?

Reply:
data, it is a habit from doing too many D3-related work, will fix